layouts/index.html

```diff
@@ -26,7 +26,7 @@ <h2 class="h4">Get started</h2>
         </div>
         <div class="col-lg-5">
           <h2 class="h4">Get details</h2>
-          <p><a href="{{ relref . "/how-to/add-code-servers" }}">Add code servers</a>, <a href="{{ relref . "/how-to/add-user" }}">invite users</a>, <br /><a href="{{ relref . "/how-to/cli" }}">setup the CLI</a>, and <a href="{{ relref . "/how-to" }}">more</a>.</p>
+          <p><a href="{{ relref . "/how-to/add-code-servers" }}">Add code servers</a>, <a href="{{ relref . "/how-to/add-user" }}">invite users</a>, <br /><a href="{{ relref . "/how-to/cli" }}">setup the CLI</a>, <a href="{{ relref . "/api" }}">dig in to the API</a>, and <a href="{{ relref . "/how-to" }}">more</a>.</p>
```

Comment:
This is unrelated to CI checks, but realized it was missing when looking up the CI API